android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -57,11 +58,6 @@
 import io.reactivex.functions.Consumer;
 import timber.log.Timber;
 
-
-/**
- * User: MananWason
```

Comment:
It not only gives a warning in the IDE and since the contribution is of all members...
Well I'll leave it be